etc/db_schema.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="inchoo_bookmark_list" resource="default" comment="Inchoo Product Bookmark List" charset="utf8" collation="utf8_general_ci">
+        <column name="bookmark_list_id" xsi:type="int" unsigned="true" identity="true" comment="Bookmark List ID" nullable="false"/>
+        <column name="bookmark_list_title" xsi:type="varchar" nullable="false" length="124" comment="Bookmark List Title"/>
+        <column name="is_deletable" xsi:type="boolean" nullable="false" default="true" comment="Is the list deletable"/>
+        <column xsi:type="int" name="customer_id" padding="10" unsigned="true" nullable="false" identity="false"
+                comment="Customer ID"/>
+        <column name="created_at" xsi:type="datetime" default="CURRENT_TIMESTAMP"/>
+        <column name="updated_at" xsi:type="datetime" default="CURRENT_TIMESTAMP" on_update="true"/>
+        <constraint xsi:type="primary" referenceId="PRIMARY">
+            <column name="bookmark_list_id"/>
+        </constraint>
+        <constraint xsi:type="foreign" referenceId="TMP" column="customer_id"
+                    referenceTable="customer_entity" referenceColumn="entity_id"
+                    onDelete="CASCADE"/>
+    </table>
+
+    <table name="inchoo_bookmark" resource="default" comment="Inchoo Product Bookmark" charset="utf8" collation="utf8_general_ci">
```

Comment:
inchoo_bookmark tablica - ja bi to nazvao inchoo_product_bookmark ili bookmark_product da bude jasnije odmah što je to

Reply:
Tablice preimenovane